package com.skysoft.vaultlogic.web.postback.mapper;

class DataMapperException extends RuntimeException {
    DataMapperException(String message, Throwable cause) {
        super(message, cause);
    }
}
